### PR TITLE
Chris/codingstandards

### PR DESCRIPTION
**Pull request by chriskavanagh97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
-# Assignment
```
Comment (harrythomas1998) [inline README.md]: Remove this

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+# Commenting and Documentation Standards
```
Comment (harrythomas1998) [inline README.md]: We use ### for these headings I believe, maybe check the master file to be sure

File: README.md (hunk 1 above)
Comment (chriskavanagh97) [reply README.md]: That is removed red lines stand for things that have been removed

File: README.md [hunk 3]
```diff
@@ -1 +1,32 @@
...
+**@TODOs** will help ensure you do not forget again. It is quite simple to implement, just mention what needs to be improved above the code, *example:*
+
+
+![Alt Text](https://lhuria94.github.io/assets/img/2017-07-26-coding-standards-that-matter/todo-example.png)
+
+
+Lastly a *“Docblock”* is a special code comment; it offers an explanation for the porpose of the code such as return values, arguments and exceptions for a block of code.
+
+
+A *“Docblock”* is a special code comment that offers an explanation of the purpose, arguments, return values, and throw exceptions for a block of code.
```
Comment (harrythomas1998) [inline README.md]: Is this line suppose to be here?

File: README.md (hunk 1 above)
Comment (harrythomas1998) [reply README.md]: Sorry didn't notice the colour

File: README.md (hunk 3 above)
Comment (chriskavanagh97) [reply README.md]: it is not I will remove

File: README.md [hunk 4]
```diff
@@ -1 +1,32 @@
-# Assignment
+# Commenting and Documentation Standards
+It is very important to add comments to your code, although it seems like a straightforward task comment **doesn’t** mean add a bunch of comments and random documentation anywhere while coding.
+
+Commenting involves mentioning the right things to help yourself and your colleagues’ when coding.
```
Comment (alinakazzaa) [inline README.md]: just colleagues without the '

**Comment by alinakazzaa** [comment]
Also, I may be wrong but you should probably pull the latest master from harry's repo so that there are fewer conflicts when merging.

File: README.md (hunk 2 above)
Comment (chriskavanagh97) [reply README.md]: changed

File: README.md (hunk 4 above)
Comment (chriskavanagh97) [reply README.md]: Will change now

**Pull request merged**